page-objects/pages/managementPages/ManageClassesPage.js

```diff
@@ -19,8 +25,45 @@ async function clickEditClassBtn() {
      await elementHelpers.clickAndWait(element);
  }
 
+ async function clickManagmentClassByNameBtn(className) {
+	await waitHelpers.waitUntilElementIsVisible(tableOfClasses);
+	let classesTable = await getClassDetailsList(classNameCell);
+	let editsElements = await elementHelpers.getListOfAllElements(editElements);
+	for (let index = 1; index <= classesTable.length; index++) {
+		let classNamePromise = await driver.$('tr:nth-child('+ index +') > td:nth-child(1)');
+        let classItem = await classNamePromise.getText();
+		if (classItem === className) {
+			await elementHelpers.clickAndWait(editsElements[index-1]);
+			break;
+		}
+    }
+}
+
+async function checkMembersInClass(className, numberOfMembers) {
+	await waitHelpers.waitUntilElementIsVisible(tableOfClasses);
+	let classesTable = await getClassDetailsList(classNameCell);
+	for (let index = 1; index <= classesTable.length; index++) {
+		let classNamePromise = await driver.$('tr:nth-child('+ index +') > td:nth-child(1)');
+        let classItem = await classNamePromise.getText();
+		if (classItem === className) {
+			let numberOfStudents = await elementHelpers.getElementText('tr:nth-child('+ index +') > td:nth-child(4)');
+            expect(numberOfMembers).to.equal(numberOfStudents)
+			break;
+		}
+    }
+}
+    // choose between className, teachers, students, schoolYear
+ async function getClassDetailsList(whichCell) {
```

Comment:
probably find another name for parameter?

Reply:
done